extensions/amp-story/1.0/page-advancement.js

```diff
@@ -508,6 +508,17 @@ export class ManualAdvancement extends AdvancementConfig {
     // <span>).
     const target = dev().assertElement(event.target);
 
+    const openLinkInNewWindow = () => {
+      target.setAttribute('target', '_blank');
+      target.setAttribute('role', 'link');
+    };
+
+    // Handles TalkBack clicks. They always come from 0 0.
```

Comment:
I'm guessing there's no other way of checking if this is TalkBack?

Reply:
Can we look at which element is focused instead? `document.activeElement`